mailbox/store/src/test/resources/eml/emailWithPDFAttachment.eml

```diff
@@ -0,0 +1,200 @@
+To: James User <user@james.org>
+From: James Sender <sender@james.org>
+Subject: Message with PDF attachment
+Message-ID: <4882e8a9-10ae-cbbe-6dbb-e17981b30be5@linagora.com>
+Date: Tue, 5 Sep 2017 10:52:35 +0200
+MIME-Version: 1.0
+Content-Type: multipart/mixed;
+ boundary="------------25E4DF31C08230523C5F2099"
+Content-Language: en-US
+
+This is a multi-part message in MIME format.
+--------------25E4DF31C08230523C5F2099
+Content-Type: text/plain; charset=utf-8; format=flowed
+Content-Transfer-Encoding: 7bit
+
+The message has a PDF attachment.
+
+--------------25E4DF31C08230523C5F2099
+Content-Type: application/pdf;
+ name="attachment.pdf"
+Content-Transfer-Encoding: base64
+Content-Disposition: attachment;
+ filename="attachment.pdf"
+
+JVBERi0xLjQKJcOkw7zDtsOfCjIgMCBvYmoKPDwvTGVuZ3RoIDMgMCBSL0ZpbHRlci9GbGF0

```

Comment:
can't we build the mail in the test code so that we have a proper pdf file in git and we can open it easily ?